### PR TITLE
feat: crowdmap

### PR DESCRIPTION
**Pull request by JoannaThelema**
Ticket: [Display Crowdmap (Mobile)](https://trello.com/c/jVYeXbfq/1854-display-crowdmap-mobile)

### CODE REVIEW

File: app/javascript/react/components/Map/Markers/CrowdMapMarkers.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      usernames,
+    ]
+  );
+
+  useEffect(() => {
+    dispatch(fetchCrowdMapData(filters));
+  }, [filters]);
+
+  useEffect(() => {
+    if (crowdMapRectangles.length > 0) {
```
Comment (evulins) [inline app/javascript/react/components/Map/Markers/CrowdMapMarkers.tsx:82]: Can we change it to const and then call it later in the useEffect?

File: app/javascript/react/components/Map/Markers/CrowdMapMarkers.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      );
+      rectanglesRef.current.push(...newRectangles);
+    }
+
+    // Cleanup function to remove rectangles on unmount
+    return () => {
+      rectanglesRef.current.forEach((rectangle) => rectangle.setMap(null));
+      rectanglesRef.current = [];
+    };
+  }, [crowdMapRectangles, map]);
```
Comment (evulins) [inline app/javascript/react/components/Map/Markers/CrowdMapMarkers.tsx:104]: We need to add ` thresholds ` here as well. Right now square colors are not changing when we adjust the threshold values.

Comment (JoannaThelema) [reply app/javascript/react/components/Map/Markers/CrowdMapMarkers.tsx:104]: Good catch! Thx! 🙌

**Review by evulins — APPROVED** [review]
Good job! 👍 I don't have any further comments regarding this matter.

**Pull request merged**